code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1892,6 +1893,96 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				H.adjustBruteLoss(LOW_PRESSURE_DAMAGE * H.physiology.pressure_mod * delta_time)
 				H.throw_alert("pressure", /atom/movable/screen/alert/lowpressure, 2)
 
+/datum/species/proc/handle_gas_interaction(mob/living/carbon/human/humi, datum/gas_mixture/environment, delta_time, times_fired)
+	if((humi?.wear_suit?.clothing_flags & STOPSPRESSUREDAMAGE) && (humi?.head?.clothing_flags & STOPSPRESSUREDAMAGE))
+		return
+
+	for(var/gas_id in environment.gases)
+		var/gas_amount = environment.gases[gas_id][MOLES]
+		switch(gas_id)
+			if(/datum/gas/antinoblium)
+				if(gas_amount > 30)
+					humi.apply_status_effect(/datum/status_effect/corrosion_curse)
+			if(/datum/gas/freon)
```

Comment:
high heat capacity gas with a strong cooling effect on most carbons, plasmamans get the opposite effect and overheat.
the gas is able to freeze the mob into an icecube if the body gets cold enough

Reply:
This feels like something better left to just the gas being cold or hot

Reply:
i kinda like the idea of iceblocking someone if it gets cold enough tbh, every gas can be hot or cold, but only freon can do this

Reply:
You should do it based on the freon temperature/the mob's bodytemp requirements rather then assuming that freon will always make you cold. This is the same freon that's used to produce the gas bomb that is hot ice